packages/dds/tree/src/feature-libraries/contextuallyTyped.ts

```diff
@@ -138,7 +138,7 @@ export function getPossibleTypes(
  * A symbol used to define a {@link MarkedArrayLike} interface.
  * @internal
  */
-export const arrayLikeMarkerSymbol: unique symbol = Symbol("editable-tree:arrayLikeMarker");
+export const arrayLikeMarkerSymbol: unique symbol = Symbol("flex-tree:arrayLikeMarker");
```

Comment:
Not sure if this could have broader consequences, would appreciate extra review on this and the other updated symbol.

Reply:
the only use-case for that string is dev facing in the debugger and similar dev stuff